diff --git a/tokenizer.py b/tokenizer.py
index 61d3a77..a9fc375 100644
--- a/tokenizer.py
+++ b/tokenizer.py
@@ -78,7 +78,7 @@ def tokenize(source: str) -> list[Token]:
                     while j < len(source) and source[j] >= "0" and source[j] <= "9":
                         num += source[j]
                         j += 1
-                num = float(num)
+                num = int(float(num))
                 is_float = True
             if not is_float:
                 num = int(num)
